mne/io/eeglab/eeglab.py

```diff
@@ -576,19 +585,74 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
         logger.info('Ready.')
 
 
-def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
-    """Create events array from EEGLAB structure.
+def read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer',
+                       uint16_codec=None):
+    r"""Create events array from EEGLAB structure.
 
     An event array is constructed by looking up events in the
     event_id, trying to reduce them to their integer part otherwise, and
     entirely dropping them (with a warning) if this is impossible.
     Returns a 1x3 array of zeros if no events are found.
+
+    Usually, the EEGLAB readers will automatically construct event information
+    for you. However, the reader for continuous data stores event information
+    in the stimulus channel, which can only code one event per time sample.
+    Use this function if your EEGLAB file has events happening at the
+    same time (sample) point to manually create an events array.
+
+    Parameters
+    ----------
+    eeg : str | object
```

Comment:
:) I think I'm not up to date with 21st century smileys yet, but no worries in any case :)